README.md

```diff
@@ -60,10 +60,14 @@ at the same time.
 
 **Primitives Library**: FuTIL can use customizable libraries based on the
 backend. The libraries live under [primitives](primitives). You probably want
-to use `primitives/std/lib` with the `-l` flag.
+to use `primitives/std/lib` with the `-l` flag, which tells the compiler to look for ```primitives/primitives/std.lib```.
```

Comment:
There's a typo in `primitives/std/lib`. It should be `primitives/std.lib`. Also, I think that it would be helpful to be explicit that `-l` passes the directory from which to search for `primitives/std.lib`. So maybe show that `-l primitives` with the import statement `import primitives/std.lib` causes the compiler to look for the file `primitives/primitives/std.lib`.

Reply:
Fixed. Tried to parallel what you're trying to say here, PTAL.

Reply:
Looks good! Thanks for doing this.